docs/docs/modules.md

```diff
@@ -0,0 +1,163 @@
+---
+layout: default
+title: KAMI Blue Modules
+description: KAMI Blue Module List
+---
+
+## KAMI Blue Modules
+
+<details>
+    <summary>Capes</summary>
+    <ul>
+        <li>Capes</li>
+        <li>LayerCape</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Combat</summary>
+    <ul>
+        <li>AntiDeathScreen</li>
+        <li>AntiFriendHit</li>
+        <li>Aura</li>
+        <li>Auto32k</li>
+        <li>AutoExp</li>
+        <li>AutoFeetPlace</li>
+        <li>AutoLog</li>
+        <li>AutoReplenish</li>
+        <li>AutoTotem</li>
+        <li>AutoTrap</li>
+        <li>BowSpam</li>
+        <li>Criticals</li>
+        <li>CrystalAura</li>
+        <li>Fastuse</li>
+        <li>Pull32k</li>
+        <li>Surround</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Experimental</summary>
+    <ul>
+        <li>GUIColour</li>
+    </ul>
+</details>
+
+<details>
+    <summary>GUI</summary>
+    <ul>
+        <li>ArmourHide</li>
+        <li>CleanGUI</li>
+        <li>InfoOverlay</li>
+        <li>InventoryViewer</li>
+        <li>Zoom</li>
+        <li>GUIColour</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Hidden</summary>
+    <ul>
+        <li>Teleport</li>
+    </ul>
+</details>
+
+<details>
+    <summary>Misc</summary>
+    <ul>
+        <li>AntiAFK</li>
+        <li>AntiChatSpam</li>
+        <li>AntiChunkBan</li>
+        <li>AntiWeather</li>
+        <li>AutoFish</li>
+        <li>AutoQMain</li>
+        <li>AutoReconnect</li>
+        <li>AutoRespawn</li>
+        <li>AutoSpawner</li>
+        <li>AutoTPA</li>
+        <li>AutoTool</li>
+        <li>BookCrash</li>
+        <li>CameraClip</li>
+        <li>ChatEncryption</li>
+        <li>ColourSign</li>
+        <li>ConsoleSpam</li>
+        <li>CustomChat</li>
+        <li>DiscordSettings</li>
```

Comment:
This is called DiscordRPC, not DiscordSettings

Reply:
> can you please move this back onto it's own page? it was fine before

bruh

Reply:
ok

Reply:
Instead of saying ok, can you fix it? If you'd like to submit a pull request don't just say ok and not make the necessary changes

Reply:
I said ok because I'm going to fix it 🤦‍♀

Reply:
That's not how github works. It shows up in the review when it's fixed, and then i can resolve it